### PR TITLE
slack webhook gateway

### PR DESCRIPTION
**Pull request by ali-graham**
screen shot 2015-06-22 at 3 27 33 pm

Tested working, and also added a simple spec for it. I'll close #877 in favour of this (I pulled the initial work from that branch).

### CODE REVIEW

File: lib/flapjack/notifier.rb [hunk 1]
```diff
@@ -120,7 +121,7 @@ def process_notification(notification)
             memo[alert] = {
               'duration' => last_change ? (Time.now.to_i - last_change) : nil,
               'state'    => ec.state,
-              'summary'  => ec.summary
+              'summary' => ec.summary
```
Comment (SarahKowalik) [inline lib/flapjack/notifier.rb]: This change looks to be unintended.

**Comment by SarahKowalik** [comment]
Looks good to me!

File: lib/flapjack/notifier.rb (hunk 1 above)
Comment (ali-graham) [inline lib/flapjack/notifier.rb]: It had been removed in the original PR, so I'd restored by re-typing. :) Will clear up once merged.

**Comment by ali-graham** [comment]
Thanks @SarahKowalik !

**Pull request merged**